### ISSUE TITLE
Integrate with Hibernate 

### ISSUE THREAD

**Issue by cheungjaidev** [SkollRyu/GUI-CRUD-SQL#1]
By any chance, the project further includes ORM?

You might want to check this out:
https://www.baeldung.com/learn-jpa-hibernate

**Comment by SkollRyu** [comment]
Yes, thank you for your advice. I would consider it as one of my to-do over the summer.
May I know how's JPA & Hibernate different from Spring / Spring Boot?
I don't quite understand the choice of them
